### PR TITLE
Replace ScholarX archive image and content

### LINKED ISSUE #1533: The ScholarX introduction paragraph and image needs refurbishing
**Is your feature request related to a problem? Please describe.**
The opening paragraph as well as the image on the ScholarX page requires refurbishing.

**Describe the solution you'd like**
Replace the paragraph with new content and replace the image with a new vector art

**Additional context**
image

**Are you working on it?**
No

### PR DESCRIPTION
**Pull request by MethmaPeiris99**
## Purpose

The purpose of this PR is to fix #1533 

## Goals

Replace the content and image of ScholarX archive

## Approach

Updated the image and the first paragraph of ScholarX page

### Screenshots

image

  
### Preview Link

https://pr-1537-sef-site.surge.sh/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: scholarx/2023/index.html [hunk 1]
```diff
@@ -48,22 +48,18 @@
...
+                drawing knowledge and expertise from all across the globe.
+              </p>
             </div>
 
-          </div>
-        </div>
-        <div class="col-md-6">
-          <img src="/assets/img/scholarx/new-findmentor.png" class="img-fluid align-self-center w-100">
+        <div class="col-md-6 align-self-center my-0">
+          <img src="./images/mentor-mentee archive/mentor-mentee archive image.png" alt="mentor-mentee archive image" class="img-fluid align-self-end w-100 px-5 py-0">
```
Comment (anjula-sack) [inline scholarx/2023/index.html]: Let rename the image to sx-hero-image @MethmaPeiris99

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Comment by janithlahirukariyawasam** [comment]
I think @MethmaPeiris99, We need to do this replacement for the ScholarX archive page.

File: scholarx/2023/index.html (hunk 1 above)
Comment (MethmaPeiris99) [reply scholarx/2023/index.html]: I'll rename it

**Comment by MethmaPeiris99** [comment]
> I think @MethmaPeiris99, We need to do this replacement for the ScholarX archive page.

Sorry for that. I'll change it @janithlahirukariyawasam

**Review by anjula-sack — APPROVED** [review]
Thank you! @MethmaPeiris99

**Pull request merged**